doc/DISCLAIMER.md

```diff
@@ -0,0 +1,3 @@
+## YunoHost specific features
+
+* Disable third party website integration with Dropbox, Google, Trello, etc
```

Comment:
It's not clear if this is meant as a "you should disable it yourself", or if the install version is the official diagramsnet but without these features ?